### ISSUE TITLE
Passing variables from one experiment to the next

### ISSUE THREAD

**Issue by AlvaroAguilera** [expfactory/expfactory-experiments#607]
Hello,

is it possible to pass some intermediate results / variable from one test to the next one when working headless? 
How about stopping the test chain / marking the token as finished according to some criteria before the normal end?

Thanks

**Comment by vsoch** [comment]
Are you looking for the experiment factor containerized version? This is for expfactory.org, which doesn't have the concept of headless.

**Comment by AlvaroAguilera** [comment]
Yes. I wasn't aware of the difference, sorry. Where is the best place to ask this question?

**Comment by vsoch** [comment]
I can transfer for you over to the expfactory repo!
